cassovary-core/src/main/scala/com/twitter/cassovary/graph/UndirectedGraph.scala

```diff
@@ -0,0 +1,29 @@
+package com.twitter.cassovary.graph
```

Comment:
pls add the standard copyright header you can get from other files (except change 2014 to 2015)